cmd/kubelet/app/options/options.go

```diff
@@ -80,46 +80,47 @@ func NewKubeletServer() *KubeletServer {
-			MaxOpenFiles:                1000000,
-			MaxPods:                     110,
-			MinimumGCAge:                unversioned.Duration{Duration: 1 * time.Minute},
-			NetworkPluginDir:            "/usr/libexec/kubernetes/kubelet-plugins/net/exec/",
-			NetworkPluginName:           "",
-			NonMasqueradeCIDR:           "10.0.0.0/8",
-			VolumePluginDir:             "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
-			NodeStatusUpdateFrequency:   unversioned.Duration{Duration: 10 * time.Second},
-			NodeLabels:                  make(map[string]string),
-			OOMScoreAdj:                 qos.KubeletOOMScoreAdj,
-			LockFilePath:                "",
-			PodInfraContainerImage:      GetDefaultPodInfraContainerImage(),
+			Address:                          "0.0.0.0",
+			CAdvisorPort:                     4194,
+			VolumeStatsAggPeriod:             unversioned.Duration{Duration: time.Minute},
+			CertDirectory:                    "/var/run/kubernetes",
+			CgroupRoot:                       "",
+			ConfigureCBR0:                    false,
+			ContainerRuntime:                 "docker",
+			CPUCFSQuota:                      true,
+			DockerExecHandlerName:            "native",
+			EventBurst:                       10,
+			EventRecordQPS:                   5.0,
+			EnableCustomMetrics:              false,
+			EnableDebuggingHandlers:          true,
+			EnableServer:                     true,
+			FileCheckFrequency:               unversioned.Duration{Duration: 20 * time.Second},
+			HealthzBindAddress:               "127.0.0.1",
+			HealthzPort:                      10248,
+			HostNetworkSources:               kubetypes.AllSource,
+			HostPIDSources:                   kubetypes.AllSource,
+			HostIPCSources:                   kubetypes.AllSource,
+			HTTPCheckFrequency:               unversioned.Duration{Duration: 20 * time.Second},
+			ImageMinimumGCAge:                unversioned.Duration{Duration: 2 * time.Minute},
+			ImageGCHighThresholdPercent:      90,
+			ImageGCLowThresholdPercent:       80,
+			LowDiskSpaceThresholdMB:          256,
+			MasterServiceNamespace:           api.NamespaceDefault,
+			MaxContainerCount:                240,
+			MaxPerPodContainerCount:          2,
+			MaxOpenFiles:                     1000000,
+			MaxPods:                          110,
+			MinimumGCAge:                     unversioned.Duration{Duration: 1 * time.Minute},
+			NetworkPluginDir:                 "/usr/libexec/kubernetes/kubelet-plugins/net/exec/",
+			NetworkPluginName:                "",
+			NonMasqueradeCIDR:                "10.0.0.0/8",
+			VolumePluginDir:                  "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
+			NodeStatusUpdateFrequency:        unversioned.Duration{Duration: 10 * time.Second},
+			NodeLabels:                       make(map[string]string),
+			OOMScoreAdj:                      qos.KubeletOOMScoreAdj,
+			LockFilePath:                     "",
+			PodInfraContainerImage:           GetDefaultPodInfraContainerImage(),
```

Comment:
I set my vim to use goimports instead of gofmt, but it does not seem to make any difference with this formatting. Also if I understand correctly, why would it because goimports does gofmt + helps with imports?